app/views/registrations/school_profile/show.html.erb

```diff
@@ -1,13 +1,15 @@
 <% content_for :before_content, govuk_back_link(text: "Back", href: registrations_question_two_path) %>
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h2 class="govuk-heading-l">What is your school Unique Reference Number(URN)?</h2>
+    <h2 class="govuk-heading-l">
+      <label for="school-profile-form-urn-field">What is your school Unique Reference Number (URN)?</label>
+    </h2>
     <%= form_with(model: @school_profile_form, url: registrations_school_profile_path) do |f| %>
       <%= f.govuk_error_summary %>
-      <label class="govuk-hint">
+      <p class="govuk-hint">
         This is a 6 digit number. Not sure? <a target='_blank' href='https://www.get-information-schools.service.gov.uk/'>Search for your school</a> on our register.
-        <%= f.govuk_text_field(:urn, width: 20, label:  nil, link_errors: true) %>
-      </label>
+      </p>
+      <%= f.govuk_text_field(:urn, width: 20, label:  nil, link_errors: true) %>
```

Comment:
I personally prefer this method: https://github.com/DFE-Digital/early-careers-framework/blob/develop/app/views/admin/suppliers/suppliers/new.html.erb#L5

Reply:
This would become 
```
<%= f.govuk_text_field(
            :urn,
            width: 20,
            label: -> { tag.h2("Supplier name", class: "govuk-heading-l") },
            hint: -> { tag.p do
              concat("This is a 6 digit number. Not sure? ")
              concat(govuk_link_to("Search for your school", "https://www.get-information-schools.service.gov.uk/"))
              concat(" on our register.")
            end
            },
            link_errors: true) %>
```

Reply:
I tried this first but was under the impression from the docs that `hint` has to be plain text only - I'll give it another go.

Reply:
Yep just tested, it's text only.

Reply:
Works for me 
![image](https://user-images.githubusercontent.com/40488007/110932809-e096b800-8323-11eb-9c85-04a97b011cf6.png)